webgui/src/flipmouse/js/tabslot.js

```diff
@@ -84,83 +36,24 @@ tabSlot.initSlots = function () {
 };
 
 tabSlot.initProcesses = function () {
-	if(inElectron === false) {
-		var resultList = ["not available / nicht verfügbar"];
-		//add the process list to the GUI
-		L('.process-select').forEach(function (elem) {
-			elem.innerHTML = L.createSelectItems(resultList);
-		});
+	if(!C.IS_ELECTRON) {
```

Comment:
if this and the `electronUtils.getProcessListForSlot(...)` line below is in the same method `tabSlot.initProcesses` it makes sense to move both into a single `if(C.IS_ELECTRON) {}` in order to keep electron-based code in one place. However then you have to set the `#processSelectionDiv` invisible by default (add `style="display: none;` to the div) and use `L.setVisible('#processSelectionDiv', true);` here.

Reply:
Done